google/ads/google_ads/config.py

```diff
@@ -196,14 +242,56 @@ def load_from_env():
         for key, env_variable in _KEYS_ENV_VARIABLES_MAP.items()
         if env_variable in os.environ
     }
-    if "logging" in config_data.keys():
+
+    specified_variable_names = config_data.keys()
+
+    # If configuration_file_path is set by the environment then configuration
+    # is retrieved from the yaml file specified in the given path.
+    if "configuration_file_path" in specified_variable_names:
```

Comment:
I know our interface is a little different from the other libraries since we have specific methods for different loading strategies (i.e. load_from_env, load_from_yaml_file), but in my mind it doesn't make sense to use this env var in the method designed to load from a yaml file and not in the method designed to load from env vars. I can use it in _both_ places, but I'd like to minimize complexity.

Reply:
I guess this comment is obsolete because it was agreed that `load_from_env` delegates to `load_from_yaml_file`.